apps/e2e/cypress/README.md

```diff
@@ -87,13 +87,13 @@ constrain them to a subset of test suite.
 Examples:
 
 ```
-# Equivalent to matching specs with `apps/e2e/integration/**/*secrets*.spec.js`
-#   - apps/e2e/integration/talentsearch/foo.secrets.spec.js
-#   - apps/e2e/integration/admin/bar.secrets.spec.js
+# Equivalent to matching specs with `apps/e2e/**/*secrets*.spec.js`
+#   - apps/e2e/talentsearch/foo.secrets.spec.js
+#   - apps/e2e/admin/bar.secrets.spec.js
 TEST_FILTER=secrets npm run e2e:run:all
 
-# Equivalent to matching spec with `apps/e2e/integration/**/*static-pages*.spec.js`
-#   - apps/e2e/integration/talentsearch/static-pages.spec.js
+# Equivalent to matching spec with `apps/e2e/**/*static-pages*.spec.js`
+#   - apps/e2e/talentsearch/static-pages.spec.js
 TEST_FILTER=static-pages npm run e2e:run:all
```

Comment:
Kinda silly but the path to these is actually `apps/e2e/cypress/e2e/**`. 

Maybe we should look into adding a ticket to not need this weird nesting of `cypress/e2e` in this app.